### PR TITLE
Bootloaders for FOTA

### PR DESCRIPTION
**Pull request by evgenibo**
Add following bootloader configurations for FOTA:

 - K64F SD
 - K66F internal flash
 - Nucleo F411RE SD
 - Nucleo F429ZI internal flash

### CODE REVIEW

File: presets/TARGET_DISCO_F769NI/TARGET_BL_INTERNAL_FLASH/mbed_app.json [hunk 1]
```diff
@@ -2,39 +2,42 @@
...
+                "|Active App Metadata Header|",
+                "+--------------------------+ <-+ mbed-bootloader.application-header-address",
+                "|                          |",
+                "|        Bootloader        |",
+                "|                          |",
+                "+--------------------------+ <-+ 0",
+                "",
+                ""
+        ]
+    }
```
Comment (jenia81) [inline presets/TARGET_DISCO_F769NI/TARGET_BL_INTERNAL_FLASH/mbed_app.json]: something is wrong with the indentation here
Where was the tdb section in the previous configuration?
Or was it just not well documented?

**Comment by jenia81** [comment]
What about Disco with external storage?
I believe we have this configuration in master

Can we somehow avoid of adding kvstore to the bootloader?
I believe we didn't have it in legacy and probably this is what adds those few KB

File: presets/TARGET_DISCO_F769NI/TARGET_BL_INTERNAL_FLASH/mbed_app.json (hunk 1 above)
Comment (evgenibo) [reply presets/TARGET_DISCO_F769NI/TARGET_BL_INTERNAL_FLASH/mbed_app.json]: In previous configuration TDB section wasn't mentioned.
I think the indentation now is correct, there was an extra tab before that.

File: presets/TARGET_K64F/TARGET_BL_SD/mbed_lib.json.template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        "*": {{
+            "target.macros_add": [
+                "MBED_CONF_STORAGE_STORAGE_TYPE=FILESYSTEM",
+                "MBED_CONF_STORAGE_FILESYSTEM_FILESYSTEM=LITTLE",
+                "MBED_CONF_STORAGE_FILESYSTEM_BLOCKDEVICE=SD",
+                "MBED_CONF_STORAGE_FILESYSTEM_EXTERNAL_BASE_ADDRESS=0x0",
+                "MBED_CONF_STORAGE_FILESYSTEM_EXTERNAL_SIZE=(1024*1024*64)",
+                "MBED_CONF_STORAGE_FILESYSTEM_INTERNAL_BASE_ADDRESS=(32*1024)",
+                "MBED_CONF_STORAGE_FILESYSTEM_RBP_INTERNAL_SIZE=(8*1024)",
+                "MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_BASE_ADDRESS=0",
```
Comment (jenia81) [inline presets/TARGET_K64F/TARGET_BL_SD/mbed_lib.json.template:13]: Internal configurations can be probably removed from this target

**Review by jenia81 — APPROVED** [review]

**Pull request merged**